doc/source/extra_content.yml.mako

```diff
@@ -1,3 +1,11 @@
+<%
+import json
+from os import environ
+from pathlib import Path
+
+with open(Path(environ['TEST_IN_DIR']) / 'test_parameters.json', 'r', encoding='utf-8') as fp:
```

Comment:
Would we rather not use python Decouple instead of pathlib? Or is pathlib python native?

Reply:
`Path` casts the string to a `Path` instance, so we can append a filename to it. `pathlib` is a Python built-in.

`os.environ` takes the environment variable. We could replace is with `decouple.config` but I don't see added value for this use case. `python-decouple` is not a dependency (yet).